tests/urlpatterns_reverse/urls_without_full_import.py

```diff
@@ -1,11 +1,11 @@
 # A URLs file that doesn't use the default
 # from django.conf.urls import *
 # import pattern.
```

Comment:
IMO this comment is obsolete, because `import *` is not a default anymore :thinking:

Reply:
Thanks :+1: